Makefile

```diff
@@ -16,11 +17,11 @@ clean:
 	rm -rf build/ dist/ src/sentry/assets.json
 	@echo ""
 
-init-config: ensure-venv
-	sentry init --dev
+init-config:
```

Comment:
So now that you've moved prerequisites over to `do.sh`, you can dedupe a lot of stuff like this:

```
develop \
init-config \
run-dependent-services \
apply-migrations \
setup-pyenv \
install-py-dev \:
    @./scripts/do.sh $@
```

Reply:
Also for `install-js-dev`, you could move `node-version-check` into lib.sh as well, then you can add `install-js-dev` to this.

Reply:
I will follow up with other changes I had in mind.